tests/src/org.hibernate.orm/hibernate-core/6.6.0.Final/src/test/java/org_hibernate_orm/hibernate_core/Target_BytecodeProvider.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_hibernate_orm.hibernate_core;
+
+import java.util.Map;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import org.hibernate.bytecode.spi.ReflectionOptimizer;
+import org.hibernate.property.access.spi.PropertyAccess;
+
+@TargetClass(className = "org.hibernate.bytecode.internal.none.BytecodeProviderImpl")
+final class Target_BytecodeProvider {
```

Comment:
@christophstrobl I see that you introduced those substitutions for testing `org.hibernate.orm` `hibernate.core`.
Why exactly are you using them and are you sure that they do not cause the test to be deviating from real-world NI use of  `org.hibernate.orm` `hibernate.core`?

Reply:
Hibernate had removed the `hibernate.bytecode.provider` option that allowed to set it to `none`.